cluster/manifests/deletions.yaml

```diff
@@ -8,4 +8,16 @@ pre_apply:
 #pre_apply: []
 
 # everything defined under here will be deleted after applying the manifests
-post_apply: []
+post_apply:
```

Comment:
We don't want to delete this in one step. We want to have these running on old nodes as discussed on the whiteboard.